src/txorphanage.h

```diff
@@ -47,6 +47,8 @@ class TxOrphanage {
      * (ie orphans that may have found their final missing parent, and so should be reconsidered for the mempool) */
     void AddChildrenToWorkSet(const CTransaction& tx, std::set<uint256>& orphan_work_set) const EXCLUSIVE_LOCKS_REQUIRED(g_cs_orphans);
 
+    size_t Size() { return WITH_LOCK(g_cs_orphans, return m_orphans.size(););};
```

Comment:
60cf71cfc16ec59857bd97608f599f8d2346c23c

1) style-nit: redundant `;` and missed space:
```suggestion
    size_t Size() { return WITH_LOCK(g_cs_orphans, return m_orphans.size()); };
```

2) what about `LOCKS_EXCLUDED(::g_cs_orphans)` thread-safety annotation?

3) style-nit: isn't `LOCK(::g_cs_orphans); return m_orphans.size();` shorter and more readable?

4) why not use global namespace `::` for global mutexes consistently?

Reply:
All fixed. Thanks for the suggestions!